config/environments/development.rb

```diff
@@ -12,13 +14,22 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  # Enable server timing
+  config.server_timing = true
+
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
   if Rails.root.join('tmp', 'caching-dev.txt').exist?
     config.action_controller.perform_caching = true
+    config.action_controller.enable_fragment_cache_logging = true
```

Comment:
Adds verbosity to logging https://edgeguides.rubyonrails.org/configuring.html#config-action-controller-enable-fragment-cache-logging